integration/key/release_name.go

```diff
@@ -0,0 +1,29 @@
+package key
+
+import (
+	"fmt"
+)
+
+func AWSConfigReleaseName(clusterID string) string {
+	return fmt.Sprintf("e2esetup-awsconfig-%s", clusterID)
```

Comment:
Seems like a mistake

Reply:
I copied the whole file from the `aws-operator`. Dropped the dead code.